dat/missions/neutral/cargo.lua

```diff
@@ -117,11 +119,14 @@ function land()
       -- Semi-random message.
       tk.msg(cargo_land_title, cargo_land_p2[rnd.rnd(1, #cargo_land_p2)]:format( cargo_land_p1[rnd.rnd(1, #cargo_land_p1)], cargo ))
       player.pay(reward)
+      dest_p, dest_s = planet.cur()
+      misn.appendLog("Cargo delivered to " .. dest_p:name() .. ", payment " .. math.floor(reward) .. " credits")
       misn.finish(true)
    end
 end
 
 function abort ()
+   misn.appendLog("Cargo delivery abandoned!")
```

Comment:
Assuming the log is supposed to be a useful reference for important past events, this is just cluttering it up with unnecessary information.